core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -172,6 +187,8 @@ public void migrate() {
               // no need to complete the job since the modification canceled the migrator job in the start event
             });
       } while (!migratorJobs.isEmpty());
+      LOGGER.info("Migration completed for Process instance with legacyId {}, the key of the migrated process is {}",
```

Comment:
🙃 Just a suggestion of phrasing here to make it less verbose.
```suggestion
      LOGGER.info("Migration completed for process instance with migrated key [{}] and legacyId [{}]",
```